src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -55,7 +55,7 @@ class Lambda {
 
     //Switch to migrations user to set permissions on it's own tables correctly
     sql"SET ROLE $migrationsUser".execute()
-    sql"GRANT SELECT, INSERT, UPDATE ON ALL TABLES IN SCHEMA public TO $apiUser;".execute.apply()
+    sql"GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO $apiUser;".execute.apply()
```

Comment:
I think this will grant delete permission to existing tables however I don't think we run this script if the user already exists.
So, if we run on fresh DB then there won't be any existing tables, I feel like it's not really useful at the moment.
Because we are giving default delete permission to new any tables.
https://github.com/nationalarchives/tdr-create-db-users/blob/ec8e9563c78a62735204d90fe7448ad0f54f5535/src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala#L63

Reply:
OK. It's just useful to have the correct scripts when needing to run this manually to fix the permission issues when cleaning out the tables. This is what I had to run to get staging to work.